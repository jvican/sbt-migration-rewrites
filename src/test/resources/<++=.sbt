lazy val target = taskKey[Seq[File]]("Target to be reassigned.")
lazy val seed = taskKey[Seq[File]]("Seed.")
seed := List(new File("."))

target <++= seed

target <++= (seed)

target <++= (seed in ThisBuild)

target <++= Def.task(seed.value)

target <++= Def.task {
  println("Executing task.")
  (seed).value
}
