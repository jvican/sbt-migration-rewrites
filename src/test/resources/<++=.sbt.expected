lazy val target = taskKey[Seq[File]]("Target to be reassigned.")
lazy val seed = taskKey[Seq[File]]("Seed.")
seed := List(new File("."))

target ++= seed.value

target ++= (seed).value

target ++= (seed in ThisBuild).value

target ++= Def.task(seed.value).value

target ++= (Def.task {
  println("Executing task.")
  (seed).value
}).value
